App crashes when a chat is opened in android 4

I installed the new version to my uncle's phone which has Android 4.4, the app crashes due to incompatibility I introduced in #1833 
seems it is this issue: 
https://stackoverflow.com/questions/8041537/how-to-reference-style-attributes-from-a-drawable

- Operating System (Linux/Mac/Windows/iOS/Android): Android 4.4
- Delta Chat Version: master
- Steps to reproduce the problem: open any chat
- Logs:
```
E/AndroidRuntime(15437): FATAL EXCEPTION: main
E/AndroidRuntime(15437): Process: chat.delta.lite, PID: 15437
E/AndroidRuntime(15437): java.lang.RuntimeException: Unable to start activity ComponentInfo{chat.delta.lite/org.thoughtcrime.securesms.ConversationActivity}: android.v
iew.InflateException: Binary XML file line #18: Error inflating class <unknown>
E/AndroidRuntime(15437):        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2205)
E/AndroidRuntime(15437):        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2255)
E/AndroidRuntime(15437):        at android.app.ActivityThread.access$800(ActivityThread.java:142)           
E/AndroidRuntime(15437):        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1203)     
E/AndroidRuntime(15437):        at android.os.Handler.dispatchMessage(Handler.java:102)                     
E/AndroidRuntime(15437):        at android.os.Looper.loop(Looper.java:136)        
E/AndroidRuntime(15437):        at android.app.ActivityThread.main(ActivityThread.java:5118)                
E/AndroidRuntime(15437):        at java.lang.reflect.Method.invokeNative(Native Method)                     
E/AndroidRuntime(15437):        at java.lang.reflect.Method.invoke(Method.java:515)                         
E/AndroidRuntime(15437):        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:792) 
E/AndroidRuntime(15437):        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:608)             
E/AndroidRuntime(15437):        at dalvik.system.NativeStart.main(Native Method)  
E/AndroidRuntime(15437): Caused by: android.view.InflateException: Binary XML file line #18: Error inflating class <unknown>                 
E/AndroidRuntime(15437):        at android.view.LayoutInflater.createView(LayoutInflater.java:620)          
E/AndroidRuntime(15437):        at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:56)            
E/AndroidRuntime(15437):        at android.view.LayoutInflater.onCreateView(LayoutInflater.java:669)        
E/AndroidRuntime(15437):        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:694)   
E/AndroidRuntime(15437):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:755)            
E/AndroidRuntime(15437):        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:814)        
E/AndroidRuntime(15437):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:745)            
E/AndroidRuntime(15437):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:758)            
E/AndroidRuntime(15437):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:758)            
E/AndroidRuntime(15437):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:758)            
E/AndroidRuntime(15437):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:758)            
E/AndroidRuntime(15437):        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:839)        
E/AndroidRuntime(15437):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:745)
E/AndroidRuntime(15437):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:758)
E/AndroidRuntime(15437):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:758)
E/AndroidRuntime(15437):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:758)
E/AndroidRuntime(15437):        at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
E/AndroidRuntime(15437):        at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
E/AndroidRuntime(15437):        at android.view.LayoutInflater.inflate(LayoutInflater.java:353)
E/AndroidRuntime(15437):        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:696)
E/AndroidRuntime(15437):        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:170)
E/AndroidRuntime(15437):        at org.thoughtcrime.securesms.ConversationActivity.onCreate(ConversationActivity.java:230)
E/AndroidRuntime(15437):        at org.thoughtcrime.securesms.PassphraseRequiredActionBarActivity.onCreate(PassphraseRequiredActionBarActivity.java:44)
E/AndroidRuntime(15437):        at android.app.Activity.performCreate(Activity.java:5275)
E/AndroidRuntime(15437):        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
E/AndroidRuntime(15437):        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2169)
E/AndroidRuntime(15437):        ... 11 more
E/AndroidRuntime(15437): Caused by: java.lang.reflect.InvocationTargetException
E/AndroidRuntime(15437):        at java.lang.reflect.Constructor.constructNative(Native Method)
E/AndroidRuntime(15437):        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
E/AndroidRuntime(15437):        at android.view.LayoutInflater.createView(LayoutInflater.java:594)
E/AndroidRuntime(15437):        ... 36 more
E/AndroidRuntime(15437): Caused by: android.content.res.Resources$NotFoundException: File res/drawable/recording_lock_background.xml from drawable resource ID #0x7f080199
E/AndroidRuntime(15437):        at android.content.res.Resources.loadDrawable(Resources.java:2136)
E/AndroidRuntime(15437):        at android.content.res.TypedArray.getDrawable(TypedArray.java:602)
E/AndroidRuntime(15437):        at android.view.View.<init>(View.java:3559)
E/AndroidRuntime(15437):        at android.view.ViewGroup.<init>(ViewGroup.java:470)
E/AndroidRuntime(15437):        at android.widget.LinearLayout.<init>(LinearLayout.java:176)
E/AndroidRuntime(15437):        at android.widget.LinearLayout.<init>(LinearLayout.java:172)
E/AndroidRuntime(15437):        ... 39 more
E/AndroidRuntime(15437): Caused by: java.lang.UnsupportedOperationException: Can't convert to color: type=0x2
E/AndroidRuntime(15437):        at android.content.res.TypedArray.getColor(TypedArray.java:327)
E/AndroidRuntime(15437):        at android.graphics.drawable.Gradien
```

--- a/res/drawable/recording_lock_background_dark.xml
+++ b/res/drawable/recording_lock_background_dark.xml
@@ -4,7 +4,7 @@
     android:shape="rectangle">
 
     <solid
-        android:color="?attr/input_panel_bg_color" />
+        android:color="@color/gray95" />
 
     <corners
         android:radius="20dp" />

--- a/res/drawable/recording_lock_background_light.xml
+++ b/res/drawable/recording_lock_background_light.xml
@@ -1,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <solid
+        android:color="@color/white" />
+
+    <corners
+        android:radius="20dp" />
+
+</shape>

--- a/res/layout/microphone_recorder_view.xml
+++ b/res/layout/microphone_recorder_view.xml
@@ -22,7 +22,7 @@
         android:layout_gravity="center"
         android:layout_marginLeft="-9dp"
         android:layout_marginStart="-9dp"
-        android:background="@drawable/recording_lock_background"
+        android:background="?attr/recording_lock_background"
         android:clipChildren="false"
         android:orientation="vertical"
         android:visibility="gone"

--- a/res/values/attrs.xml
+++ b/res/values/attrs.xml
@@ -70,6 +70,7 @@
     <attr name="emoji_variation_selector_background" format="reference|color" />
     <attr name="quick_camera_icon" format="reference"/>
     <attr name="quick_mic_icon" format="reference"/>
+    <attr name="recording_lock_background" format="reference"/>
 
     <attr name="conversation_item_background" format="reference"/>
     <attr name="conversation_item_incoming_bubble_color" format="reference|color"/>

--- a/res/values/themes.xml
+++ b/res/values/themes.xml
@@ -147,6 +147,7 @@
 
         <item name="quick_camera_icon">@drawable/quick_camera_light</item>
         <item name="quick_mic_icon">@drawable/ic_mic_grey600_24dp</item>
+	<item name="recording_lock_background">@drawable/recording_lock_background_light</item>
 
         <item name="conversation_item_sent_indicator_text_background">@drawable/conversation_item_sent_indicator_text_shape</item>
 
@@ -305,6 +306,7 @@
 
         <item name="quick_camera_icon">@drawable/quick_camera_dark</item>
         <item name="quick_mic_icon">@drawable/ic_mic_white_24dp</item>
+	<item name="recording_lock_background">@drawable/recording_lock_background_dark</item>
 
         <item name="menu_new_conversation_icon">@drawable/ic_add_white_24dp</item>
         <item name="menu_search_icon">@drawable/ic_search_white_24dp</item>

